src/Arcus.Scripting.ActiveDirectory/Scripts/Add-AzADAppRoleAssignment.ps1

```diff
@@ -38,11 +38,11 @@ try {
         $adApplication.AppRole += $newRole
 
         Update-AzADApplication -ObjectId $adApplication.Id -AppRole $adApplication.AppRole
-        Write-Host "Added role '$Role' to Active Directory Application '$($adApplication.DisplayName)'"
+        Write-Host "Added role '$Role' to Active Directory Application '$($adApplication.DisplayName)'" -ForegroundColor White
```

Comment:
I think it's a good idea to have a look at how things are logged, and use different loglevels.
But, it also makes me think:  if we do this, why do we stick with `Write-Host` instead of using `Write-Information` ?

https://stackoverflow.com/questions/38523369/write-host-vs-write-information-in-powershell-5

OTH, I've been playing a bit with it right now, but when I execute a Write-Information command in my terminal, nothing is displayed.  Maybe the default loglevel is set to warning ...